app/src/main/res/values/strings.xml

```diff
@@ -953,6 +953,8 @@
     <string name="share_link_all_apps_subheader">All actions</string>
     <!-- Sub-header in the dialog to share a link to an app from the most-recent sorted list -->
     <string name="share_link_recent_apps_subheader">Recently used</string>
+    <!-- An string shown when an account is signed in where %1$s is a placeholder for the email-->
+    <string name="sync_signed_as">Signed in as %1$s</string>
     <!-- An option from the share dialog to sign into sync -->
     <string name="sync_sign_in">Sign in to Sync</string>
```

Comment:
Just confirming this is capital S "Sync" not lowercase 😁

Reply:
Good catch, the design spec shows it as lowercase! Changing it now :)